app/Policies/UserPolicy.php

```
@@ -0,0 +1,48 @@
+    /**
+     * Determine whether the user can update the model.
+     */
+    public function update(User $user, User $model): bool
+    {
+        return $user->isAdmin() || $user->is($model);
+    }
+
+    /**
+     * Determine whether the user can delete the model.
+     */
+    public function delete(User $user, User $model): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$model'. (undefined)

(UnusedFormalParameter)